Add possibility to disable installation of dependencies (#50)
insightsengineering/thevalidatoR: README.md

  _Required_: `false`

  _Default_: `v1`

* `disable_install_dev_deps`:

>   _Description_: Disable installation of dev dependencies during running report.

Reviewer: ```suggestion
  _Description_: Disable installation of dev dependencies while building the report.
```
